pkgs/applications/editors/neovim/gnvim/wrapper.nix

```diff
@@ -12,9 +12,16 @@ stdenv.mkDerivation {
       --prefix PATH : "${neovim}/bin" \
       --set GNVIM_RUNTIME_PATH "${gnvim-unwrapped}/share/gnvim/runtime"
   '' else ''
+    gappsWrapperArgs+=(
```

Comment:
Thumbnailers are probably not necessary. They would be used by `libgnome-desktop`, which this does not seem to depend on.